giskard/client/giskard_client.py

```diff
@@ -122,6 +122,29 @@ def get_project(self, project_key: str) -> Project:
         response = self._session.get("project", params={"key": project_key}).json()
         return Project(self._session, response["key"], response["id"])
 
+    def delete_project(self, project_key: str) -> str:
+        """
+        Function to delete the project that belongs to the mentioned project key
+        Args:
+            project_key:
+                The unique value of  project provided during project creation
+        """
+        analytics.track("Delete Project", {"project_key": anonymize(project_key)})
+        try:
+            response = self._session.get("project", params={"key": project_key}).json()
+            project_id = response["id"]
+            self._session.delete(f'project/{project_id}')
```

Comment:
Could you add a project deletion method on a java side that would take a project key directly instead of fetching an `id` and then using it to delete the project by id?

Reply:
Yes, will do that instead.